qa/tasks/vstart_runner.py

```diff
@@ -187,10 +187,14 @@ def wait(self):
         out, err = rm_nonascii_chars(out), rm_nonascii_chars(err)
         if isinstance(self.stdout, StringIO):
             self.stdout.write(out.decode(errors='ignore'))
+        elif self.stdout == None:
```

Comment:
```suggestion
        elif self.stdout is None:
```

as there is only a single instance of `None` in Python, instead of checking the equality, normally, people check the identity when checking if an instance **is** None.

Reply:
Right. I keep switching back to using `==` subconsciously.

Reply:
Fixed. Thanks!